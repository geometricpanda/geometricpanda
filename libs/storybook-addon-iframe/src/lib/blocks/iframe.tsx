import { styled } from '@storybook/theming'

export const IFrame = styled.iframe`
  background-color: white;
  border: 0;
  height: 100%;
  width: 100%;
`;
