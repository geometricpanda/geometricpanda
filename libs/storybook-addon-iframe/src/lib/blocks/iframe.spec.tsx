import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { IFrame } from './iframe';


describe('iFrame', () => {
  it('should snapshot', () => {
    const tree = renderer.create(<IFrame src='https://www.google.com' />);
    expect(tree).toMatchSnapshot();
  });
});
