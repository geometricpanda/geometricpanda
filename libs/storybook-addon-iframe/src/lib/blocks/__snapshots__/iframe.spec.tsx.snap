// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`iFrame should snapshot 1`] = `
.c0 {
  background-color: white;
  border: 0;
  height: 100%;
  width: 100%;
}

<iframe
  className="c0"
  src="https://www.google.com"
/>
`;
